apps/researcher/src/app/[locale]/objects/[id]/date-created-formatter.ts

```diff
@@ -0,0 +1,21 @@
+import {TimeSpan} from '@/lib/api/definitions';
+
+export function dateCreatedFormatter(dateCreated: TimeSpan, locale: string) {
```

Comment:
Nice solution :) Just one detail: in the code base we do use names ending with `..ter` (e.g. `LabelFetcher`), but as far as I can see not for functions. Use a verb instead, for consistency - e.g. `formatDateCreated()`?